tests/conftest.py

```diff
@@ -2665,22 +2665,22 @@ def updated_default_storage_class_ocs_virt(
         )
         == "false"
     ):
-        failed_update_boot_sources = False
+        boot_source_imported_successfully = False
         with remove_default_storage_classes(cluster_storage_classes=cluster_storage_classes):
             with update_default_sc(default=True, storage_class=ocs_storage_class):
-                failed_update_boot_sources = re_import_boot_sources(
```

Comment:
if update_default_sc fails , i know chances are usually less , then there will be no default sc left in cluster?

Reply:
if there will be no default sc in the cluster - boot source images will not be ready, so it will fail anyway

Reply:
but if we catch early we will reduce wait time and fail right away.? wdyt